pyxlsb/reader.py

```diff
@@ -144,7 +144,7 @@ def read_id(self):
     v = 0
     for i in range(4):
       byte = self._fp.read(1)
-      if byte == '':
+      if byte == b'':
```

Comment:
Just for the sake of keeping Python 2 compat in the 1.0.x line, simplifying the check will support both cases:
```suggestion
      if not byte:
```